tiatoolbox/models/models_abc.py

```diff
@@ -15,24 +15,26 @@ class IOConfigABC(ABC):
     @property
     @abstractmethod
     def input_resolutions(self):
+        """Abstract method to update input_resolution."""
         raise NotImplementedError
 
     @property
     @abstractmethod
     def output_resolutions(self):
+        """Abstract method to update output_resolutions."""
         raise NotImplementedError
 
 
 class ModelABC(ABC, nn.Module):
     """Abstract base class for models used in tiatoolbox."""
 
-    def __init__(self):
+    def __init__(self) -> None:
+        """Initializes Abstract class ModelABC."""
```

Comment:
```suggestion
        """Initialize Abstract class ModelABC."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090